package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ng-bootstrap/ng-bootstrap",
-  "version": "6.1.0",
+  "version": "7.0.0",
```

Comment:
Oh, could you please also not modify the version for now? Leave at `6.1.0`

Reply:
Reverted to 6.1.0